indigo_api/serializers.py

```diff
@@ -326,6 +332,19 @@ def validate(self, attrs):
             if doc.namespace != AKN_NAMESPACES['3.0']:
                 raise ValidationError(f"Document must have namespace {AKN_NAMESPACES['3.0']}, but it has {doc.namespace} instead.")
 
+        if attrs.get('content') and attrs.get('provision_eid'):
+            # validate the portion content
+            try:
+                xml = attrs['content']
+                ns = AKN_NAMESPACES[DEFAULT_VERSION]
+                xml = f'<akomaNtoso xmlns="{ns}">{xml}</akomaNtoso>'
```

Comment:
rather get the client to provide this as fully valid AKN, so the server doesn't have to make guesses